src/ctim/schemas/common.cljc

```diff
@@ -23,7 +23,18 @@
 
 (def ctim-schema-version "1.0.8")
 
-(def-eq CTIMSchemaVersion ctim-schema-version)
+(cs/def ::ctim-schema-version
+  #(re-matches #"\w+.\w+\.\w+" %))
+
+(def SchemaVersion
+  (f/str
+   :description (str "A semantic version matching the CTIM version against which "
+                     "this object should be valid.")
+   :default ctim-schema-version
+   :gen (cs/gen #{ctim-schema-version})
+   :spec (cs/and string? ::ctim-schema-version)))
+
+(def CTIMSchemaVersion SchemaVersion)
```

Comment:
Why did you change the schema of the clim-schema-version? For me the value for each CTIM version if a static value.